en/full-node.md

```diff
@@ -1107,10 +1107,47 @@ automatically started minimized in the task bar.
 #### Bitcoin Core Daemon {#osx-daemon}
 {:.no_toc}
 
-If you can provide instructions and screenshots for running the latest
-version of Bitcoin Core daemon on OS X Yosemite, please [open an
-issue](https://github.com/bitcoin-dot-org/bitcoin.org/issues/new) and we'll tell
-you what we need.
+The Bitcoin Core daemon (bitcoind) is not included in the .dmg file you may have downloaded to install Bitcoin-QT. Bitcoind, along with its support binaries, is instead included in the OS X .tar.gz file listed on the official Bitcoin Core download page. To download this file using Terminal, execute the following command:
+
+    curl -O https://bitcoin.org/bin/bitcoin-core-0.11.0/bitcoin-0.11.0-osx64.tar.gz
+
+{{verifyReleaseSignatures}}
+
+Extract bitcoind and its support binaries from the archive we just downloaded by running this command in Terminal:
+
+    tar -zxvf bitcoin-0.11.0-osx64.tar.gz
```

Comment:
This is a suggestion, not a request, but I always prefer to omit the -v (verbose) flag from tar so that I see any errors instead of having them buried in a flood of output.  The Bitcoin tarball should be small enough that the command will return almost immediately, so users won't be sitting there wondering, "is it doing anything?"